modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -489,6 +489,10 @@ const setup = (editor: Editor, setupForTheme: ThemeRenderSetup): RenderInfo => {
     });
 
     editor.addQueryStateHandler('ToggleToolbarDrawer', () => OuterContainer.isToolbarDrawerToggled(outerContainer));
+
+    editor.on('blur', (_e) => {
+      OuterContainer.toggleToolbarDrawerWithoutFocusing(outerContainer);
```

Comment:
Shouldn't you check `isToolbarDrawerToggled` first? otherwise it might _show_ the toolbar when the editor loses focus? 😉 

(if that's a real bug in this change, add a test for it)